### DISCUSSION TITLE
Navidrome set up help

### DISCUSSION THREAD

**Discussion by Atc1** [navidrome/navidrome#2790] [Q&A]
I recently installed Navidrome on Ubuntu virtual machine hosted at Digital ocean and I was able to transfer files using FileZilla. It is working fine but I need help on how to fix the following issues please:

1, How can I change the login page to my website's name and logo and add login info for users

2, How to restrict guest users from changing password

3, How can I connect my domain to Navidrome 

5, On my computer album covers are shown but when I upload an album to Navidrome the album cover is missing 

6, Songs are not numbered when they are written with different alphabet

![navidrome login page](https://github.com/navidrome/navidrome/assets/75685091/1c345818-f493-454b-9b01-d27594c1a8f2)
![navidrome guest user](https://github.com/navidrome/navidrome/assets/75685091/dad17f53-7b28-4a9b-8bd7-fa2e1e57ffee)
![Navidrome album cover](https://github.com/navidrome/navidrome/assets/75685091/5bbaf68b-d0fb-4757-915d-f3489d4ef5a2)
![Songs numbering](https://github.com/navidrome/navidrome/assets/75685091/39405a48-a935-4bf3-99a6-a42e698fb071)

**Comment by jsaathof** [comment]
I don't have answers to all your questions. I am also a new Navidrome user.

> 1, How can I change the login page to my website's name and logo and add login info for users

There is a message you can set with "UIWelcomeMessage" in your config file. I have not found on how to change the icon. I can understand that you cannot change the icon. There is an option to set the background ("UILoginBackgroundUrl")

> 2, How to restrict guest users from changing password

This can be turned off by setting "EnableUserEditing" to false in your config file.

> 3, How can I connect my domain to Navidrome

I don't understand what you mean with this question. What do you want to do exactly?

> 5, On my computer album covers are shown but when I upload an album to Navidrome the album cover is missing

Are there errors in the logging about the album covers? If you have only embedded album covers you could try to set the  "CoverArtPriority" option to "embedded" in your config file. Or you can set the LastFM credentials in your config file to get the external album covers. I only have folder.jpg files and that works. Are you sure the album covers are actually embedded in the files?

> 6, Songs are not numbered when they are written with different alphabet

I don't think this is true. I believe that this is read from the metadata (tags) of the file. Can you check if the numbering is present in the files?

**Reply by Atc1** [reply]
Thank you for your response. For question #3 what I meant was that how to use a custom domain ( mydomain.com) instead of using the default IP address 159.89.33.176:4533
Thanks.

**Comment by jsaathof** [comment]
For that you will need a DNS service and register your IP address. This is not some setting in Navidrome.

**Reply by Atc1** [reply]
Hi, When I installed Navidrome I used a YouTube tutorial video that helped me install it and not sure if I created a config file. How can I access the config file by the way? Like what is the Ubuntu command line to open it?
